staging/src/k8s.io/kubectl/pkg/config/types.go

```diff
@@ -16,7 +16,10 @@ limitations under the License.
 
 package config
 
-import metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+import (
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	clientcmdapi "k8s.io/client-go/tools/clientcmd/api"
```

Comment:
it's really confusing to have kuberc types referencing clientcmd types ... I'd suggest defining these locally in staging/src/k8s.io/kubectl/pkg/config/types.go and staging/src/k8s.io/kubectl/pkg/config/v1beta1/types.go, and translate them into internal clientcmd policy types in one spot in applyPluginPolicy

Reply:
To be clear, are you asking to essentially duplicate the types between kuberc and clientcmd?

Reply:
If that is correct, what options are available to provide guarantees that the types won't drift apart over time?